src/cider/piggieback.clj

```diff
@@ -206,9 +206,15 @@
               (apply merge
                      ((juxt :requires :require-macros)
                       (ana/get-namespace ana/*cljs-ns*)))]
-      (reader/read {:read-cond :allow :features #{:cljs}}
-                   (readers/source-logging-push-back-reader
-                    (java.io.StringReader. form-str))))))
+      (let [rdr (readers/source-logging-push-back-reader
```

Comment:
I think you should also document this in the docstring of the function just so it's clear to everyone how this is supposed to behave.